<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterOptions</class>
 <widget class="QWidget" name="ParameterOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <item row="4" column="0" alignment="Qt::AlignTop">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="listLvlParameter"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QTableWidget" name="rulesTable">
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Наименование</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Отношение</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" alignment="Qt::AlignTop">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Значение</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Уровень</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameParameter"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Правила</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Тип данных</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Ограничения</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="listTypeDParameter"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="valParameter"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QTableWidget" name="restrictionTable">
     <column>
      <property name="text">
       <string>Наименование</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Отношение</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QTextEdit" name="descriptionParameter"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Наименование</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="editParameter_Button">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Применить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
